ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2585,8 +2585,8 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-gcp-upi-xpn
-- as: ibmcloud-ipi-p2-f7
-  cron: 12 11 6,14,21,28 * *
+- as: ibmcloud-ipi-p3-f28
```

Comment:
The same case in 4.13, I think we also can update the ibmcloud-ipi-p2-f7 job in 4.13.